app/assets/javascripts/Components/Modals/download_test_results_modal.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import Modal from "react-modal";
+
+class DownloadTestResultsModal extends React.Component {
+  constructor(props) {
```

Comment:
You can remove the constructor if you aren't doing anything other than calling the super class one.